apps/api/cmd/flow/main_test.go

```
@@ -0,0 +1,277 @@
+			if err := command.Process.Signal(test.signal); err != nil {
+				t.Fatal(err)
+			}
+			done := make(chan error, 1)
+			go func() { done <- command.Wait() }()
+			select {
+			case err := <-done:
+				var exitError *exec.ExitError
+				handled := errors.As(err, &exitError) && exitError.ExitCode() == 2
+				terminated := false
+				if errors.As(err, &exitError) {
+					if status, ok := exitError.ProcessState.Sys().(syscall.WaitStatus); ok {
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "ProcessState" from selector

(staticcheck)